### PR TITLE
Switch datasource for dropdowns to `v2/linkables` endpoint

### PR DESCRIPTION
**Pull request by elliotcm**
https://trello.com/c/GmaspITI/617-modify-policy-publisher-to-use-linkables-endpoint

See commit messages for more detail.

### CODE REVIEW

File: spec/presenters/content_item_presenter_spec.rb [hunk 1]
```diff
@@ -73,5 +73,13 @@
       expect(nation_applicability["alternative_policies"].first["nation"]).to eq("northern_ireland")
       expect(nation_applicability["alternative_policies"].first["alt_policy_url"]).to eq("https://www.example.ni")
     end
+
+    it "includes the internal name if the policy is a sub-policy" do
```
Comment (Davidslv) [inline spec/presenters/content_item_presenter_spec.rb]: I would suggest to create a context 

```ruby
context "when policy is a sub-policy" do
  it "includes the internam name" do
    ...
```

**Comment by Davidslv** [comment]
@elliotcm there's some tests failing, could you have a look into it please?
Thank you

**Comment by elliotcm** [comment]
@Davidslv being looked into, thanks

**Comment by tuzz** [comment]
I didn't understand the `internal_name` thing, but that's because I don't really know Policy Publisher.

This pull request looks fine to me. :+1:

**Pull request merged**